LayoutTests/platform/mac-wk2/fast/dom/navigator-detached-no-crash-expected.txt

```diff
@@ -11,6 +11,7 @@ navigator.contacts is OK
 navigator.cookieEnabled is OK
 navigator.credentials is OK
 navigator.hardwareConcurrency is OK
+navigator.identity is OK
```

Comment:
`identity` is also showing up in mac-wk1, unfortunately. I need to figure out how to turn that off. Any suggestions?